src/Settings/SettingsAttribute/SearchableAttribute.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/**
+ * This file is part of AlgoliaSearch Client PHP Helper.
+ *
+ * (c) Algolia Team <contact@algolia.com>
+ *
+ *  For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ */
+
+namespace Algolia\AlgoliaSearch\Helper\Settings\SettingsAttribute;
+
+use Algolia\AlgoliaSearch\Helper\Contracts\SettingContract;
+use Algolia\AlgoliaSearch\Helper\Helpers\Str;
+
+/*
+ * @internal
+ */
+final class SearchableAttribute implements SettingContract
+{
+    /**
+     * @var string[]
```

Comment:
Can you use `array<int, string>`? Please also check in all source code if we are using this annotations.